x-pack/plugins/stack_connectors/server/connector_types/cases/jira/service.ts

```diff
@@ -136,6 +136,9 @@ export const createExternalService = (
     }, '');
   };
 
+  const escapeSpecialCharacters = (str: string) =>
+    str.replace(/[!^&*()+\-[\]\\/{}|:?~]/g, '\\\\$&')?.replace(/-/g, '\\\\x2d');
```

Comment:
Do we need to escape unicode? I don't see it initially from searching this site: https://confluence.atlassian.com/jirasoftwareserver/search-syntax-for-text-fields-939938747.html

It'd be worth some manual testing though.

Reply:
I don't think we need it. The `encodeURIComponent` will encode the URL correctly.